src/mft/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+## This script is to build the kernel-mft-dkms_4.5.0-3.16.0-4-amd64 kernel modules
+##
+## USAGE:
+##   ./build.sh
+
+MFT_NAME=mft-4.5.0-31-x86_64-deb
+MFT_TGZ=${MFT_NAME}.tgz
+MFT_KERNEL_DEB=kernel-mft-dkms_4.5.0-31_all.deb
+KERNELVER=3.16.0-4-amd64
+
+[ -e $MFT_TGZ ] || wget http://www.mellanox.com/downloads/MFT/${MFT_TGZ}
+tar xzf $MFT_TGZ
+pushd $MFT_NAME/SDEBS
+dpkg -i $MFT_KERNEL_DEB
+TARBALL_PATH=$(dkms mkdriverdisk kernel-mft-dkms/4.5.0 -d ubuntu -k ${KERNELVER} --media tar | grep "Disk image location" | cut -d':' -f2)
```

Comment:
dkms mkdriverdisk command creates debian package with architecture x86_64, which yields an error during instalation due to arch mismatch
You can add option -a all to set architecture type to all to allow any arch